shell.nix

```diff
@@ -0,0 +1,49 @@
+let
+  pkgs = (import <nixpkgs> { });
```

Comment:
Can you write down an example nixpkgs version that works for you here? I just tried a couple and they both failed (for different reasons).

Reply:
I meant even getting nix-shell to succeed here, but I was able to get it working since (my nix version needed an update).